devfile/script.md

```diff
@@ -1,9 +1,30 @@
 # Devfile Demo
 
 ## Introduction
+TODO
 
-## Steps
+## Script (timing ~15 minutes)
 
-- Create a nodjs mongo workspace using a devfile with 2 docker images components with a Factory
-- Edit devfile to use a Kubernetes component with a yaml reference. Create a new workspace
-- Start workspace from existing Kubernetes deployment
\ No newline at end of file
+- Deploy Node JS application with Mongo
+  - Demonstrate content of [deploy_k8s.yaml](deploy_k8s.yaml)
+  - Execute
+    * `oc new-project nodejs-app`
+    * `oc apply -f deploy_k8s.yaml`
+  - Demo that application works fine. On minishift/minikube it should be available at http://nodejs.192.168.99.100.nip.io/
```

Comment:
With minishift, you could also substitute `http://nodejs.$(minishift ip).nip.io/` here, since the actual ip address varies between instances

Reply:
Well, to be k8s complaint I use ingress but not Route and host are hard-coded there. Will take a look and maybe will do it openshift specific, and then it will be able to put here a command to fetch application host from route.

Reply:
Hm I thought that the ingress would also be created on the VM's IP. I'll double check.